### DISCUSSION TITLE
Windows release error when reading analysis from file

### DISCUSSION THREAD

**Discussion by dawidza** [dundee/gdu#127] [Q&A] [answered]
Firstly, thanks for such an awesome tool. I am getting the below error when reading analysis from file using the latest Windows release. Any idea what I might be doing wrong?

No items to display.                                                                                                   panic: runtime error: invalid memory address or nil pointer dereference
                                                                       [signal 0xc0000005 code=0x0 addr=0x68 pc=0x38ae3f]

 goroutine 35 [running]:
                        github.com/dundee/gdu/v5/pkg/analyze.(*Dir).GetPath(0xc001dded00)
                                                                                                /home/dundee/work/gdu/pkg/analyze/file.go:168 +0x7f
                           github.com/dundee/gdu/v5/tui.(*UI).ReadAnalysis.func1()
                                                                                        /home/dundee/work/gdu/tui/actions.go:118 +0x135
               created by github.com/dundee/gdu/v5/tui.(*UI).ReadAnalysis
                                                                                /home/dundee/work/gdu/tui/actions.go:103 +0x956

**Comment by dundee** [comment]
Looks like the analysis is not read correctly. Could you paste the import file here?

**Comment by dawidza** [comment]
Here you go. I have zipped it. Usually import it unzipped. Thanks for taking a look.

[report.zip](https://github.com/dundee/gdu/files/8025008/report.zip)

**Reply by dundee** [reply]
Thanks, it breaks for me as well. I will investigate what's wrong.

**Reply by dawidza** [reply]
Thank you very much.

**Comment by dundee** [comment, marked as answer]
Fix just released in 5.13.1. Thanks for reporting!
